app/src/main/java/app/revanced/integrations/youtube/patches/components/AdsFilter.java

```diff
@@ -100,6 +104,16 @@ public AdsFilter() {
                 "expandable_list"
         );
 
+        channelProfile = new StringFilterGroup(
+                null,
+                "channel_profile.eml"
+        );
+
+        visitStoreButton = new ByteArrayFilterGroup(
+                Settings.HIDE_VISIT_STORE_BUTTON,
+                "header_store_button"
```

Comment:
@LisoUseInAIKyrios It may be enough to just filter for this without an additional check for channelProfile. Should we get rid of channelProfile?